lib/test/Test/Pos/Cbor/CborSpec.hs

```diff
@@ -363,6 +389,270 @@ spec = withDefConfiguration $ do
             pendingNoArbitrary "EdStandard.SecretKey"
             pendingNoArbitrary "EdStandard.Signature"
             pendingNoArbitrary "EncryptedSecretKey"
+        describe "Primitive instances" $ do
+            binaryTest @()
+            binaryTest @Bool
+            binaryTest @Char
+            binaryTest @Integer
+            binaryTest @Word
+            binaryTest @Word8
+            binaryTest @Word16
+            binaryTest @Word32
+            binaryTest @Word64
+            binaryTest @Int
+            binaryTest @Float
+            binaryTest @Int32
+            binaryTest @Int64
+            binaryTest @Nano
+            binaryTest @Millisecond
+            binaryTest @Microsecond
+            binaryTest @Byte
+            binaryTest @(Map Int Int)
+            binaryTest @(HashMap Int Int)
+            binaryTest @(Set Int)
+            binaryTest @(HashSet Int)
+
+        describe "Types" $ do
```

Comment:
AFAIU, all these tests below are now inside `modifyMaxSuccess (const 1000)`, but previously they weren't. I think we shouldn't change these limits in this PR (or at least it should be done carefully, I am not sure it was).